migrations/20200927132044-update-patient-barcode-non-nullable.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.sequelize.query(
+          `UPDATE patients SET "barcodeValue" = "id" WHERE "barcodeValue" IS NULL`,
+          { transaction: t }
+        ),
+        queryInterface.sequelize.query(
+          'ALTER TABLE patients ALTER COLUMN "barcodeValue" TYPE "varchar", ALTER COLUMN "barcodeValue" SET NOT NULL',
+          { transaction: t }
+        ),
```

Comment:
[there is an existing sequelize method that can alter a column without a raw query](https://sequelize.org/master/class/lib/dialects/abstract/query-interface.js~QueryInterface.html#instance-method-changeColumn) - is it possible to use this instead? this seems straightforward enough that we can use a sequelize method, which is preferable as they _should_ be more consistent than raw queries (same deal for the equivalent `down` query)